aiida/restapi/resources.py

```diff
@@ -192,6 +193,7 @@ def __init__(self, **kwargs):
         self.utils_confs = {k: kwargs[k] for k in utils_conf_keys if k in
                             kwargs}
         self.utils = Utils(**self.utils_confs)
+        self.method_decorators = {'get': kwargs.get('get_decorators', [])}
```

Comment:
Ok, so is there a good reason why Node does not inherit from BaseResource?
The explanation below "class Node(Resource)" is not entirely clear to me, perhaps you could make it a docstring and expand it a little bit?

Reply:
NodeResource class is different from BaseResource in trans.set_query() mostly because it takes query_type as an input and the presence of additional result types like "tree". I have updated the doc string.